skbio/alignment/_pairwise.py

```diff
@@ -795,9 +838,8 @@ def _init_matrices_nw_no_terminal_gap_penalty(
 
 
 def _compute_substitution_score(aln1_chars, aln2_chars, substitution_matrix,
```

Comment:
Yes, it is used in `_compute_score_and_traceback_matrices` which is used by `global_pairwise_align` and `local_pairwise_align`.